### DISCUSSION TITLE
Avira Password Manager makes input fields lose focus

### DISCUSSION THREAD

**Discussion by PieterxJan** [livewire/livewire#5083] [1. Help]
Hi,

We have a form with a few input fields. Some have the `.lazy` option added to the `wire:model` attribute. 

But there are also a few fields that should trigger a search in a table. Now we've recently discovered that, when people have the Avira Password Manager activated in their browser, the input field loses focus once it makes the call to the server. 

I was wondering whether there is anything we could do on our end to prevent this from happening?

### How it should be (Password manager disabled)
![good_example](https://user-images.githubusercontent.com/1104660/177000145-8d7a2f93-4a56-4472-9b4d-7143fbbf63d2.gif)

### How it is (Password manager enabled)
This might not come across very good in the gif, but a click is needed to regain focus.

![bad_example](https://user-images.githubusercontent.com/1104660/177000123-70de80bc-ae03-4ea7-bcf5-76c385b1637f.gif)

Kind regards
PJ

**Comment by Martleby** [comment]
Might be worth trying one of the Javascript lifecycle hooks (element.updated) to set the focus after Livewire has finished updating the DOM.

Cheers,
Martin.

**Comment by devront-it** [comment]
Same problem here. Are there any ideas? Re-focusing the input is not really an option, because sometimes the user wants to edit a text field in the middle of its content. So re-focusing would result in cursor jumping around.
